### PR TITLE
Predict fix

### PR DESCRIPTION
**Pull request by PiotrNawrot**

### CODE REVIEW

File: trax/models/research/funnel_transformer_test.py [hunk 1]
```diff
@@ -230,49 +230,52 @@ def test_funnel_transformer_lm_forward_shape_eval(self):
...
+        d_model=d_model, d_ff=d_model, n_heads=n_heads,
         vanilla_attn_type=attention_type,
         mode='eval')
 
-    rng_1 = jax.random.PRNGKey(0)
-
-    x = np.zeros((batch_size, n_len_eval)).astype(np.int32)
-    _, _ = eval_funnel.init(shapes.signature(x), rng=rng_1, use_cache=False)
-    y_eval = eval_funnel(x)
+    input = np.random.randint(low=0, high=vocab_size,
```
Comment (syzymon) [inline trax/models/research/funnel_transformer_test.py]: prefer jax.random with fixed seed

**Review by syzymon — APPROVED** [review]
